### DISCUSSION TITLE
Running in CSP environment

### DISCUSSION THREAD

**Discussion by cAstraea** [opcodesio/log-viewer#359] [Ideas]
Had a previous issue but that was fixed on my end sorry about it. However trying it in another project found it's incompatible with our csp settings ? 

Uncaught EvalError: Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'

Is it possible to run it without unsafe-eval ? Seems the weback is generating a 
const render = new Function(“Vue”, code)(_vue_runtime_dom__WEBPACK_IMPORTED_MODULE_0__); 

Have any ideas on how to do this without adding the unsafe-eval to our headers ? 
Did some investigation and seems there's an option for using the vue-runtime build ? Not sure if this would fix the issue though 
https://github.com/laravel-mix/laravel-mix/issues/1052

**Comment by arukompas** [comment]
Hey @cAstraea 

I tried previously to switch to a runtime build but was unsuccessful at it. Some things just broke and it was not usable.

It can be quite time consuming to do this, so I can't make any promises here. If you ever do attempt it, a PR is always welcome! 🙏
